packages/trader/src/AppV2/Components/Guide/Description/video-fragment.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Localize } from '@deriv/translations';
+import { Loading } from '@deriv/components';
+import { Text } from '@deriv-com/quill-ui';
+import { useDevice } from '@deriv-com/ui';
+import { getUrlBase } from '@deriv/shared';
+
+type TVideoFragment = {
+    contract_type: string;
+};
+const VideoFragment = ({ contract_type }: TVideoFragment) => {
+    const [is_loading, setIsLoading] = React.useState(true);
+
+    const { isMobile } = useDevice();
+
+    // memoize file paths for videos and open the modal only after we get them
+    const getVideoSource = React.useCallback(
+        (extension: string) =>
+            getUrlBase(`/public/videos/${contract_type.toLowerCase()}_${isMobile ? 'mobile' : 'desktop'}.${extension}`),
+        [contract_type, isMobile]
+    );
+    const mp4_src = React.useMemo(() => getVideoSource('mp4'), [getVideoSource]);
+
+    return (
+        <div
+            className={classNames('video-fragment__wrapper', {
+                'video-fragment__wrapper--accumulator': contract_type.toLowerCase() === 'accumulators',
```

Comment:
Unfortunately, I was not able to use `const CONTRACT_LIST` from utils in order not to hardcode `'accumulators'` because it creates a circular dependency (`VideoFragment` component)

Reply:
Please move `getContractDescription` to a different file so that we won't circular dependency?